dist/0.1.6/global.js

```diff
@@ -1,203 +1,7 @@
-/* nebraskajs - v0.1.6 - 2015-04-03
+/* nebraskajs - v0.1.6 - 2015-04-23
```

Comment:
I think you may need to pull in master or run an npm install or something here. It’s missing some content.